kod/object/active/holder/nomoveon/battler/player.kod

```diff
@@ -7663,7 +7663,7 @@ messages:
          }
 
          % Give newbies a little bonus.
-         if piBase_Max_health < 26 and random(20,bound(piBase_Max_health,20,25)) = 20
+         if piBase_Max_health = PKILL_ENABLE_HP
```

Comment:
This only gives the bonus if the player's max heal is equal to PKILL_ENABLE_HP. We have this on 103:

     if piBase_Max_health < Send(Send(SYS,@GetSettings),@GetPKillEnableHP)

So you would want to use:

    if piBase_Max_health < PKILL_ENABLE_HP